### PR TITLE
Infinality

### PR DESCRIPTION
**Pull request by thypon**
#1053 #1257

### CODE REVIEW

File: srcpkgs/freetype/template [hunk 1]
```diff
@@ -12,6 +12,30 @@ license="GPL-2"
 distfiles="${SOURCEFORGE_SITE}/$pkgname/$pkgname-$version.tar.bz2"
 checksum=387bcc4b780b12484aa2ec9f7db1a55d8286eb5639f45fbc0fbba7a4e5a1afb9
 
+build_options="ultimate"
+desc_option_ultimate="Enable ultimate patchset"
+build_options_default="ultimate"
```
Comment (xtraeme) [inline srcpkgs/freetype/template]: I will refuse to merge this if this is enabled by default.

Comment (Gottox) [inline srcpkgs/freetype/template]: What do you think patched packages {cairo,freetype,fontconfig}-ultimate which provide {cairo,freetype,fontconfig}?

**Comment by xtraeme** [comment]
I won't maintain anything related to those unofficial patches, just FYI.

File: srcpkgs/freetype/template (hunk 1 above)
Comment (xtraeme) [inline srcpkgs/freetype/template]: This will cause more harm than good.

**Comment by thypon** [comment]
I could help mantaining them; i <3 perfect font rendering. An alternative could be using the ubuntu patchset. @xtraeme do you prefer them enabled by default?

**Comment by thypon** [comment]
Added steam-fonts to fix a missrendering with ultimate patchset.

**Comment by skrzyp** [comment]
:+1:  Respect for your effort - that weird way of compiling those patches always took off me from making their templates.
FYI, not only you love those perfect font rendering, especially with [kori's config](https://github.com/kori/shell.d/blob/master/infinality)

**Comment by thypon** [comment]
@xtraeme Given is disableable i'd prefer to include these option by default. All the major desktop distribution has his own patchset to render correctly fonts, but i think that infinality is the more cross distribution patchset / configurable approach.

**Comment by skrzyp** [comment]
@thypon : What's going on with this port? Maybe we should provide is at alternative package which can replace fontconfig, instead of making fontconfig flavor?

Sorry for asking too much, but I just can't wait for it ^^

**Comment by thypon** [comment]
You can clone and use it right now like i'm doing since a week waiting that @xtreame decide something.

Right now is disabled by default then you need to enable with the "-oultimate" flag.

```
git clone https://github.com/thypon/void-packages
cd void-packages/
git checkout infinality
./xbps-src binary-bootstrap
xbulk -oultimate cairo fontconfig freetype steam-fonts
xbps-install --repository=hostdir/binpkgs/infinality/ cairo fontconfig freetype steam-fonts
```

I really don't want to mantain a not so used/out of sync alternative package like in archlinux. I really hope we could merge this one as default or use the patchset from fedora/ubuntu to obtain a good font rendering per default out of the box.

**Comment by skrzyp** [comment]
Thank you so much, @thypon!
Maybe you should convince @xtraeme by providing additional infos about it and screenshots?
Also, you could hang out with us on IRC :>

**Comment by thypon** [comment]
Merged manually

**Comment by Duncaen** [comment]
Infinality is unmaintained and was dropped from void-packages, fontconfig improved and merged parts of it.

**Pull request closed without merging**